spec/capitalize_decorator.rb

```diff
@@ -0,0 +1,13 @@
+require 'rspec'
+require_relative '../capitalize_decorative'
+
+RSpec.describe CapitalizeDecorator do
+  let(:nameable) { double('Decorator', correct_name: 'john doe') }
+  let(:decorator) { CapitalizeDecorator.new(nameable) }
+
+  describe '#correct_name' do
+    it 'capitalizes the name provided by the nameable object' do
+      expect(decorator.correct_name).to eq('John Doe') # Capitalize "John" and "Doe"
+    end
+  end
+end
```

Comment:
- [ ] Please fix the errors in this class test as shown below in the screenshot. 🙏 
![capitalize_decorator](https://github.com/faisalakb/Ruby_OOP_school_library/assets/47625407/3eb7050d-34c4-486e-879f-754689833968)